actors/miner/tests/util.rs

```diff
@@ -309,18 +309,54 @@ impl ActorHarness {
 
         let mut info = Vec::with_capacity(num_sectors);
         for pc in precommits {
-            let sector = self.prove_commit_sector_and_confirm(
-                rt,
-                &pc,
-                self.make_prove_commit_params(pc.info.sector_number),
-                ProveCommitConfig::empty(),
-            );
+            let sector = self
+                .prove_commit_sector_and_confirm(
+                    rt,
+                    &pc,
+                    self.make_prove_commit_params(pc.info.sector_number),
+                    ProveCommitConfig::empty(),
+                )
+                .unwrap();
             info.push(sector);
         }
         rt.reset();
         info
     }
 
+    pub fn commit_and_prove_sector(
+        &self,
+        rt: &mut MockRuntime,
+        sector_no: SectorNumber,
+        lifetime_periods: i64,
```

Comment:
nit: for a better name than golang tests go with lifetime_period or lifetime_epochs

Reply:
Then `commit_and_prove_sectors` should probably be updated as well. I'll leave it for another PR.